monero-outreach-docs/translations/fr/archive/triptych_fr.md

```diff
@@ -0,0 +1,43 @@
+
+Les algorithmes du "core ring signature" ont également été améliorés: la publication de proposition originelle ("whitepaper") de CryptoNote présentait un processus de signature de cercle différent de celui que nous employons actuellement, qui est bâti sur l’utilisation des signatures "Multilayered Linkable Spontaneous Anonymous Group" (MLSAG). Une amélioration substantielle, nommée signatures "Compact Linkable Spontaneous Anonymous Group" (CLSAG), est attendue prochainement, et promet de réduire d’environ un quart le poids (en termes de données) des transactions et d’un dixième leur temps de confirmation (pour une transaction de deux entrées et deux sorties).
+
+Toutes ces améliorations ont donné à Monero de puissantes capacités d’anonymisation. A chaque mouvement de Monero, tout TXO sortant prend la forme d’une clé publique TXO de couverture, sans lien cryptographique avec l’adresse émettrice, mêlée a 10 autres clé publiques-leurres, sélectionnées pour leur vraisemblance. Cela rend toute surveillance extrêmement difficile, tout en assurant une rapidité et une fluidité d’exécution du logiciel de Monero, et offrant une chaine de blocs de taille et de vitesse acceptable.
+
+### _Les Défis_
+
+Néanmoins, la technologie obfusquant l’émetteur n’est pas optimale. Dix transactions-leurres ne sont pas toujours suffisantes parce que certains TXO aléatoires pourraient être identifiés comme étant des leurres. Ils auraient déjà pu servir à des transactions identifiées précédemment. L’existence de listes noires pourrait poser problème et offrir à des adversaires motivés une arme potentielle en prenant l’initiative de les ajouter à la liste de leurs portefeuilles. Il y a également d’autres astuces pour identifier les leurres, en liant les TXO aux échanges ou aux pools de mineurs de Monero, en estimant leur usage dans d’autres transferts et en évaluant des probabilités statistiques temporelles.
+
+Imaginons une séance d'identification au poste de police, composée d’un individu accusé et de 10 autres personnes (dans cet exemple, disons que ce sont des policiers habillés en civil), intégrées à la séance afin d'éviter que l'accusé ne soit identifié automatiquement et ainsi s'assurer de la véracité de l'identification. Si les policiers sont facilement identifiables par le témoin, cette stratégie ne serait d’aucun recours à l’individu. Il en va de même pour les signatures de cercle de Monero: la technique de se fondre dans la foule ne peut marcher que lorsque la foule est convaincante et l'identité des participants n'est pas connue par les tiers. Contre des adversaires déterminés, 10 leurres choisis au hasard peuvent ainsi paraître insuffisants. Pourquoi, alors, ne pas en intégrer plus?
```

Comment:
Original sentence:
Imagine a police lineup with one wrongly accused and 10 decoys. If the decoys were all known to a false witness, it wouldn’t work to protect the accused. 
Suggested sentence:
Imaginons une séance d'identification au poste de police, composé d'un innocent et de 10 autres accusés. Si les 10 accusés sont tous des amis d'un supposé témoin, il ne sera pas possible de protéger la personne innocente et celle ci risque d'être déclarée coupable, bien qu'elle soit innocente. 

Reason: The original english text is of very poor quality and it is hard to translate it accurately. But I think my suggestion is closer to the original: The proposed translation departs very significantly from the original text

Reply:
We had discussed this in a separate channel, and your suggested sentence is both closer to the original translation and more concise. Integrated.